services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/table/TableQueryManagerImpl.java

```diff
@@ -94,6 +94,8 @@ public QueryResultBundle querySinglePage(ProgressCallback progressCallback, User
 			if (options.runQuery()) {
 				rowHandler = new SinglePageRowHandler();
 			}
+
+			String combinedSql = createCombinedSql(user, query, options.returnCombinedSql());

```

Comment:
String combinedSql = null
if(option.returnCombindedSql()){
   combindedSQl= create()
}
unconditionally set the bundle.